win32/updateBuildLibs.py

```diff
@@ -265,6 +251,9 @@ def main():
         usage()
 
     print('Updating source by %s branch.' % branch)
+    if not os.path.exists(LOG_PATH):
+        os.makedirs(LOG_PATH)
+
```

Comment:
We put everyday's build logs in the same log_path. Now I need to make this log_path with time stamp, so we can keep our logs dated and avoid over write the previous day's log. So I make this log_path as global and create it at the start point main() before we do update and build.